src/python/pants/option/global_options.py

```diff
@@ -475,860 +486,764 @@ def from_options(cls, options: OptionValueContainer) -> LocalStoreOptions:
-            type=float,
-            default=60.0,
-            help="The maximum amount of time to wait for the invocation to start until "
-            "raising a timeout exception. "
-            "Because pantsd currently does not support parallel runs, "
-            "any prior running Pants command must be finished for the current one to start. "
-            "To never timeout, use the value -1.",
-        )
-        register(
-            "--pantsd-max-memory-usage",
-            advanced=True,
-            type=memory_size,
-            default="1GiB",
-            # `memory_size` will convert the `default` to # bytes, but we want to keep the string
-            # for `./pants help`.
-            default_help_repr="1GiB",
-            help=(
-                "The maximum memory usage of the pantsd process.\n\n"
-                "When the maximum memory is exceeded, the daemon will restart gracefully, "
-                "although all previous in-memory caching will be lost. Setting too low means that "
-                "you may miss out on some caching, whereas setting too high may over-consume "
-                "resources and may result in the operating system killing Pantsd due to memory "
-                "overconsumption (e.g. via the OOM killer).\n\n"
-                "You can suffix with `GiB`, `MiB`, `KiB`, or `B` to indicate the unit, e.g. "
-                "`2GiB` or `2.12GiB`. A bare number will be in bytes.\n\n"
-                "There is at most one pantsd process per workspace."
-            ),
-        )
-
-        # These facilitate configuring the native engine.
-        register(
-            "--print-stacktrace",
-            advanced=True,
-            type=bool,
-            default=False,
-            help="Print the full exception stack trace for any errors.",
-        )
-        register(
-            "--engine-visualize-to",
-            advanced=True,
-            default=None,
-            type=dir_option,
-            help=(
-                "A directory to write execution and rule graphs to as `dot` files. The contents "
-                "of the directory will be overwritten if any filenames collide."
-            ),
-        )
-
-        # Pants Daemon options.
-        register(
-            "--pantsd-pailgun-port",
-            advanced=True,
-            type=int,
-            default=0,
-            daemon=True,
-            help="The port to bind the Pants nailgun server to. Defaults to a random port.",
-        )
-        register(
-            "--pantsd-invalidation-globs",
-            advanced=True,
-            type=list,
-            default=[],
-            daemon=True,
-            help="Filesystem events matching any of these globs will trigger a daemon restart. "
-            "Pants's own code, plugins, and `--pants-config-files` are inherently invalidated.",
-        )
+class BootstrapOptions:
+    """This are the set of options necessary to create a Scheduler. If an option is not consumed
```

Comment:
FYI generally we try to have the first paragraph be a single sentence. Also I think the grammar is off on first sentence:

```suggestion
    """The set of options necessary to create a Scheduler.
    
    If an option is not consumed...
```